### ISSUE TITLE
Microsoft.Extensions.Logging instead of Common.Logging

### ISSUE THREAD

**Issue by adamlith** [splitio/.net-core-client#71]
Would you consider switching over to the more modern Microsoft.Extensions.Logging library for logging instead of Common.Logging? It has the same capabilities and more, and is also much nicer to work with.

**Comment by NicoZelaya** [comment]
Hi @adamlith, thank you for reaching out!

We'll take a look at it with our .Net team, test it and aim to update it in the upcoming releases. After a quick look it makes sense to update and use Microsoft.Extensions.Logging.

In any case, thank you very much for flagging! We'll keep this issue open so we can let you know here once we make the update.

Best,
Nico.

**Comment by AdamSchueller** [comment]
@NicoZelaya any updates on this issues?

This was a pain point for us as well -- we had to create a shim for Common.Logging to get it to work with our existing services, which have all been updated to use the Core logging abstractions.

We have a TODO in the code to monitor this issue and remove the extra code once this is resolved.

**Comment by NicoZelaya** [comment]
Hi @AdamSchueller ! 

I'm sorry to hear the pain this caused to you and your team. It's already in our backlog waiting for prioritization from our PM. 
We'll keep you posted here once we have a target date, we'll tackle it probably on the next couple of sprints.

Regards,
Nico.

**Comment by sanzmauro** [comment]
Hi @adamlith and @AdamSchueller !

We released 6.0.0 and in this version we started to use `Microsoft.Extensions.Logging` library.
So to enable the split logs you need you need use our extension method in your `ILoggerFactory` inside your Startup class:

```c#
//...
using Microsoft.Extensions.Logging;

public class Startup
{
    //...
    public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
    {
        //...
        loggerFactory.AddSplitLogs();
    }
}
```

For more information please visit our Documentation: https://help.split.io/hc/en-us/articles/360020240172--NET-SDK

In this version we did a breaking change, so we don't support `Common.Logging` for our Net Core SDK anymore.

Please let me know if you have any concerns after upgrading the sdk.
My apologies for the delay in this notification.

Kindest regards,
Mauro.
